src/relion/__init__.py

```diff
@@ -30,10 +31,18 @@ def __init__(self, path):
         :param path: A string or file system path object pointing to the root
                      directory of an existing Relion project.
         """
+        super().__init__("Import/job001")
         self.basepath = pathlib.Path(path)
         if not self.basepath.is_dir():
             raise ValueError(f"path {self.basepath} is not a directory")
-        self.pipeline = RelionPipeline("Import/job001")
+        try:
+            self.load()
+        except Exception:
```

Comment:
overly broad?

Reply:
Yes, probably. It is to catch the case where the file that contains the Relion pipeline information is missing so that the Project doesn't just become completely useless. I've change it to catch `FileNotFoundError`